.gitattributes

```diff
@@ -0,0 +1 @@
+start.sh eol=lf
```

Comment:
@DerStimmler please insert the new line at the end.

Reply:
It's the only line in the file.
Or do you mean an empty line at the end of the file?

Reply:
@FrankStrieter I updated the PR.